tests/draft2020-12/dynamicRef.json

```diff
@@ -756,5 +756,29 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description" : "json pointer in $dynamicRef",
+        "schema":{
+            "$schema":"https://json-schema.org/draft/2020-12/schema",
```

Comment:
We already have this covered. See line 715.

Reply:
Although the test on 715 is a full URI with fragment; so this does ensure you are picking up the correct root when only a fragment is supplied. I would say the description is wrong, but it is not necessarily a duplicate test. That said, it feels more like one of those extended "implementation" tests (e.g. we don't have tests for syntactically borked URIs because the baseline assumption is that you are following the underlying rules for URIs, numbers etc.)

Reply:
I also thing this is a duplicate. See line 134.